src/uuid.cr

```diff
@@ -129,12 +129,12 @@ struct UUID
     new(new_bytes, variant, version)
   end
 
-  def self.empty
+  def self.empty : self
     new(StaticArray(UInt8, 16).new(0_u8), UUID::Variant::NCS, UUID::Version::V4)
   end
 
   # Returns UUID variant.
-  def variant
+  def variant : UUID::Variant
```

Comment:
No need to use full paths.

```suggestion
  def variant : Variant
```

Reply:
I think it's better as is. It's not returning a "Variant", it's returning a UUID of Variant kind

Reply:
That's a general naming problem we have. The type should have been called `VariantUUID`